src/scrabble.js

```diff
@@ -1,5 +1,48 @@
-function scrabble() {
-  // write code here
+const scrabbleValue = {
+  A: 1,
+  E: 1,
+  I: 1,
+  O: 1,
+  U: 1,
+  L: 1,
+  N: 1,
+  R: 1,
+  S: 1,
+  T: 1,
+  D: 2,
+  G: 2,
+  B: 3,
+  C: 3,
+  M: 3,
+  P: 3,
+  F: 4,
+  H: 4,
+  V: 4,
+  W: 4,
+  Y: 4,
+  K: 5,
+  J: 8,
+  X: 8,
+  Q: 10,
+  Z: 10
+}
+
+const scrabble = (makeWord) => {
+  if (
+    makeWord === `    ` ||
+    makeWord === ` \t\n` ||
+    makeWord === null ||
+    typeof makeWord !== `string`
+  ) {
+    return 0
+  }
+  const printWord = makeWord.toUpperCase()
```

Comment:
I like your naming convention, using words to clearly state what everything is so you can keep track through your code!